forge/test/mlir/test_ops.py

```diff
@@ -49,6 +49,51 @@ def forward(self, x):
 ]
 
 
+@pytest.mark.parametrize(
+    "params",
+    [
+        ([(1, 256, 24, 24), (1, 256, 24, 24)], -4),
+        ([(5, 64, 128, 128), (5, 64, 128, 128)], -3),
+        ([(1, 30, 30, 16), (1, 30, 30, 16)], -2),
+        ([(1, 30, 30, 16), (1, 30, 30, 16)], 3),
+        ([(5, 64, 128, 128), (5, 64, 128, 128)], -1),
+        ([(1, 256, 24, 24), (1, 256, 24, 24)], 4),
+        ([(1, 256, 24, 24), (1, 256, 24, 24)], 2),
+        ([(5, 64, 128, 128), (5, 64, 128, 128)], 1),
+        ([(1, 30, 30, 16), (1, 30, 30, 16)], 0),
+    ],
+)
+def test_stack(params):
+    _, _, dim = params[0][0], params[0][1], params[1]
+    if dim == -3 or dim == 1:
+        pytest.xfail("Tensor rank is not 4")
```

Comment:
Is this failing on the FFE side?

Reply:
It is failing in run_mlir_compiler stage
log: [stack_mlir.log](https://github.com/user-attachments/files/17616626/stack_mlir.log)